modules/git/blame.go

```diff
@@ -23,12 +24,16 @@ type BlamePart struct {
 
 // BlameReader returns part of file blame one by one
 type BlameReader struct {
-	cmd            *Command
 	output         io.WriteCloser
 	reader         io.ReadCloser
 	bufferedReader *bufio.Reader
 	done           chan error
 	lastSha        *string
+	ignoreRevsFile *os.File
```

Comment:
It seems strange to store a closed `File` here. Maybe using `ignoreRevsFileName string` is good enough?

Otherwise it's not easy to understand when the file should be closed.

Reply:
I used `os.File` here to have an explicit `nil` check. Sure a `!= ""` check works too but my intent was to have a real file linked to the object. How about changing this member to `os.FileInfo`? That should be the best of both worlds.

Reply:
> but my intent was to have a real file linked to the object. 

But the file has been closed, the object itself can't do other things except accessing the `f.Name()`, which is simply `{ return f.name }`